x/ecocredit/server/basket/query_basket.go

```diff
@@ -24,5 +25,28 @@ func (k Keeper) Basket(ctx context.Context, request *baskettypes.QueryBasketRequ
 		return nil, err
 	}
 
-	return &baskettypes.QueryBasketResponse{Basket: basketGogo}, nil
+	it, err := k.stateStore.BasketClassStore().List(ctx, basketv1.BasketClassPrimaryKey{}.WithBasketId(basket.Id))
+	if err != nil {
+		return nil, err
+	}
+
+	var classes []string
+	for it.Next() {
+		class, err := it.Value()
+		if err != nil {
+			return nil, err
+		}
+
+		basketClassGogo := &baskettypes.BasketClass{}
+		err = PulsarToGogoSlow(class, basketClassGogo)
+		if err != nil {
+			return nil, err
+		}
+
+		classes = append(classes, basketClassGogo.ClassId)
```

Comment:
```suggestion
		classes = append(classes, class.ClassId)
```
don't need to convert to gogo